seed/audit_template/audit_template.py

```diff
@@ -64,3 +95,56 @@ def get_api_token(self):
             raise validation_client.ValidationClientException(f"Expected JSON response from Audit Template: {response.text}")
 
         return response_body['token'], ""
+
+
+@shared_task
+def _get_buildings(cycle_id, url, headers):
+    try:
+        response = requests.request("GET", url, headers=headers)
+        if response.status_code != 200:
+            return None, f'Expected 200 response from Audit Template get_buildings but got {response.status_code}: {response.content}'
+    except Exception as e:
+        return None, f'Unexpected error from Audit Template: {e}'
+    at_buildings = response.json()
+    result = []
+    for b in at_buildings:
+        # Only update properties that have been recently updated on Audit Template
+        at_updated = datetime.fromisoformat(b['updated_at']).strftime("%Y-%m-%d %I:%M %p")
```

Comment:
Audit Template returns iso format dates for the `updated` attribute. SEED formats updated/created as `YYYY-MM-DD HH:mm:ss p`. I think we should keep date formats consistent across seed